src/config.js

```diff
@@ -22,5 +22,5 @@ export const discordConfig = {
 
 export const version = package_json.version;
 
-export const POST_CHANNEL_ID = "1245319554201157724"
+export const POST_CHANNEL_ID = process.env.POST_CHANNEL_ID.toString() || ""
```

Comment:
**question:** Can the application run without channel ID? Should that be optional?

Reply:
Yes, can be optional. Discord bots continues to run after throwing API req errors.